packages/grid/x-data-grid/src/constants/defaultGridSlotsComponents.ts

```diff
@@ -42,7 +41,11 @@ import {
   GridDragIcon,
   GridColumnHeaderFilterIconButton,
   GridSearchIcon,
+  GridVisibilityOffIcon,
+  GridViewColumnIcon,
+  GridClearIcon,
 } from '../components';
+import { GridColumnMenuDefault } from '../components/reexportable';
```

Comment:
```suggestion
import { GridColumnMenuDefault } from '../components/menu/columnMenu/default/GridColumnMenuDefault';
```
To avoid importing simple menu components as well